software/emacs/README.md

```diff
@@ -0,0 +1,5 @@
+[Emacs](https://www.gnu.org/software/emacs/) has existed from the dawn of time.
+
+![real_programmers_use_emacs](./real_programmers.png)
```

Comment:
Is this link broken?

Reply:
@catheraaine - Which link isn't working for you?